pkg/alerts/policies.go

```diff
@@ -137,3 +241,64 @@ type alertPolicyResponse struct {
 type alertPolicyRequestBody struct {
 	Policy Policy `json:"policy"`
 }
+
+type alertQueryPolicyCreateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+
+type alertQueryPolicyUpdateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+type alertQueryPolicyResponse struct {
+	Actor struct {
+		Account struct {
+			Alerts struct {
```

Comment:
Seems like `Alerts` should be the struct, then some GraphQL wrapper removal around it.  We should think of the best way to do this as it's an ongoing issue.  Ideally, we'll get to code-gen at some point and refactor all of these out.

Reply:
I see we do this for `data` currently, but is `Actor.Account` as prolific in our GQL responses?  I suppose we could case switch on the client, but then we're putting package level logic into the client.  Is there another approach you were thinking  of here?

Reply:
`actor` will always be there, it's the scope the request was made under to NerdGraph.  `Account` is an entire section of the NerdGraph schema.

This is fine for now, we need to look at a better way to generate structs from GQL schema so we aren't constantly building these out.